### PR TITLE
Add host-kernel cfg

### PR DESCRIPTION
**Pull request by suqinhuang**
Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
id: 1292022

File: shared/cfg/host-kernel/host-kernel-check.cfg [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        requires_kernel = [">= 2.6.32-505", "<  2.6.32-574"]
+            - 7:
+                host_kernel_ver_str += ".7"
+                variants:
+                    - 0:
+                        host_kernel_ver_str += ".0"
+                        requires_kernel = [">= 3.2.1-0.8", "< 3.10.0-124"]
+                    - 1:
+                        host_kernel_ver_str += ".1"
+                        requires_kernel = [">= 3.10.0-124", "< 3.10.0-230"]
```
Comment (CongSmile) [inline shared/cfg/host-kernel/host-kernel-check.cfg]: How about RHEL.7.2 ?

**Comment by suqinhuang** [comment]
@CongSmile @xutian updated, could you review it again

File: shared/cfg/host-kernel/Host_RHEL/5/9.cfg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- 9:
+    host_kernel_ver_str += ".9"
+    requires_kernel = [">= 2.6.18-348", "<  2.6.19"]
```
Comment (CongSmile) [inline shared/cfg/host-kernel/Host_RHEL/5/9.cfg]: should this be <2.6.18-349 instead of 2.6.19 ?

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by xutian** [comment]
looks fine, merged thanks @suqinhuang and @CongSmile

**Pull request merged**